index.js

```diff
@@ -35,6 +36,13 @@ function init(editor) {
 			editor.setTabLength(config.tab_width);
 		}
 	}
+
+	if (config.charset) {
+		// editorconfig charsets names matches iconv charsets names (that is used by Atom)
+		// so no charset name convertion is needed.
+		editor.setEncoding(config.charset);
```

Comment:
Nothing special happens: Atom's file charset is set to this unsupported value, user see that the file is edited in an unknown encoding, reads the README.md and correct it. All Atom packages assume user is a dev and know howto read the doc.

Of course there's a TODO here: add a warning to user to let him know an unsupported encoding is given in .editorconfig file; but for now there is no messages to the user in the package.